test/tests/jobs/RequeueJobsTest.cpp

```diff
@@ -195,4 +196,31 @@ struct RequeueJobsTest : tpunit::TestFixture {
         ASSERT_EQUAL(result[0][0], "QUEUED");
         ASSERT_EQUAL(result[0][1], "newJobName");
     }
+
+    void testNextRunTime() {
+        // Some time setup
+        const uint64_t time = STimeNow();
+        string oldTime = SComposeTime("%Y-%m-%d %H:%M:%S", time - 10'000'000);
```

Comment:
this and current time are the same value, why do you need both? also, since you can combine these into one object, why do you need the `time` variable on the line above this?

Reply:
Whoops, that was supposed to have been flipped on this line, fixing